pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -124,7 +124,7 @@ type CRDDescription struct {
 // APIServiceDescription provides details to OLM about apis provided via aggregation
 // +k8s:openapi-gen=true
 type APIServiceDescription struct {
-	Name              string                 `json:"name"`
+	GenerateName      string                 `json:"generateName"`
```

Comment:
Don't we want to change the field in `WebhookDescription`, not `APIServiceDescription`?

Reply:
Resolved.